lockbox-ios/View/AccountSettingView.swift

```diff
@@ -70,16 +70,22 @@ extension AccountSettingView: UIGestureRecognizerDelegate {
         ]
 
         let leftButton = UIButton()
-        leftButton.setTitle(Constant.string.settingsTitle, for: .normal)
-        leftButton.setTitleColor(.white, for: .normal)
-        leftButton.setTitleColor(Constant.color.lightGrey, for: .selected)
-        leftButton.setTitleColor(Constant.color.lightGrey, for: .highlighted)
-
-        let backImage = UIImage(named: "back")
+        let backImage = UIImage(named: "back")?.withRenderingMode(.alwaysTemplate)
+        let tintedBackImage = backImage?.tinted(UIColor(white: 1.0, alpha: 0.6))
         leftButton.setImage(backImage, for: .normal)
-        leftButton.adjustsImageWhenHighlighted = false
+        leftButton.setImage(tintedBackImage, for: .selected)
+        leftButton.setImage(tintedBackImage, for: .highlighted)
+        leftButton.setTitle(Constant.string.settingsTitle, for: .normal)
 
+        leftButton.contentHorizontalAlignment = .left
+        leftButton.setContentCompressionResistancePriority(.defaultLow, for: .horizontal)
         leftButton.titleEdgeInsets = UIEdgeInsets(top: 0, left: 5, bottom: 0, right: -20)
+        leftButton.setTitleColor(.white, for: .normal)
+        leftButton.setTitleColor(UIColor(white: 1.0, alpha: 0.6), for: .highlighted)
+        leftButton.setTitleColor(UIColor(white: 1.0, alpha: 0.6), for: .selected)
+        leftButton.setTitleColor(UIColor(white: 1.0, alpha: 0.6), for: .disabled)
+        leftButton.tintColor = .white
+        leftButton.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Whoops, I think this unnecessary addition was the problem. After removing:

<img width="299" alt="screen shot 2018-05-21 at 4 15 32 pm" src="https://user-images.githubusercontent.com/49511/40332744-74a3bd6c-5d12-11e8-93b1-630d819853d2.png">

<img width="252" alt="screen shot 2018-05-21 at 4 17 20 pm" src="https://user-images.githubusercontent.com/49511/40332748-76b16a96-5d12-11e8-97db-750d0a99acad.png">